addons/hr_org_chart/static/src/fields/hr_org_chart.js

```diff
@@ -31,25 +32,9 @@ class HrOrgChartPopover extends Component {
         this.rpc = useService('rpc');
         this.orm = useService('orm');
         this.actionService = useService("action");
-    }
-
-    /**
-     * Get subordonates of an employee through a rpc call.
-     *
-     * @private
-     * @param {integer} employee_id
-     * @returns {Promise}
-     */
-    async _getSubordinatesData(employee_id, type) {
-        const subData = await this.rpc(
-            '/hr/get_subordinates',
-            {
-                employee_id: employee_id,
-                subordinates_type: type,
-                context: Component.env.session.user_context,
-            }
-        );
-        return subData;
+        onWillRender(() => {
+            this._onEmployeeSubRedirect = onEmployeeSubRedirect.bind(this);
```

Comment:
Instead of binding here, you can have the hook to return a function, like it's done in [hr_holidays] for example.
In your hook you can call `useService()`

[hr_holidays]: https://github.com/odoo/odoo/blob/master/addons/hr_holidays/static/src/views/hooks.js#L8

```suggestion
        this.onEmployeeSubRedirect = onEmployeeSubRedirect;
```